gms/api/src/main/pegasus/com/linkedin/dataset/Dataset.pdl

```diff
@@ -122,4 +122,14 @@ record Dataset includes DatasetKey, ChangeAuditStamps, VersionTag {
    * List of global tags applied to the dataset
    */
   globalTags: optional GlobalTags
+
+  /**
+   * Related business term of the dataset
+   */
+  datasetGlossaryTerm: optional DatasetGlossaryTerm
```

Comment:
I think this model is a bit off- the way this stands you can only apply 1 term to the dataset at a time. Is that intentional? I would think we would want to be able to apply an arbitrary number of terms.

The struct name is also more specific than it needs to be. This property type should just be `GlossaryTerms`- we shouldnt need an `<Entity>GlossaryTerms` struct for every entity.

You can refer to how we modeled tags as reference. That supports arbitrary tags applied at the entity and field level

Reply:
I'd also advise taking a look at how we apply tags to the SchemaField and EditableSchemaField struct for consistency of glossary term > schema field application. We apply tags inside the schema field struct so all schema field metadata is co-located. Then, to allow for multiple writers we also created an EditableSchemaMetadata struct.

Reply:
will be making those changes